.github/workflows/build_and_test.yml

```diff
@@ -25,7 +25,7 @@ jobs:
         yes | sudo dpkg -i pygplates_package.deb
     - name: Install gplately
       run: |
-        pip install stripy cartopy==0.19.0.post1
```

Comment:
@brmather do you remember why we required this specific version? It comes from here: 0adcc64

Reply:
I think @lauren-ilano found a bug with shapely 2 and cartopy v0.21, but it looks like this has been managed.